frontend/src/components/hub/CustomBackground.tsx

```diff
@@ -0,0 +1,119 @@
+import { Theme, useMediaQuery } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import React, { useContext } from "react";
+import UserContext from "../context/UserContext";
+
+const useStyles = makeStyles((theme) => ({
+  background: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    zIndex: -1,
+    overflow: "hidden",
+  },
+
+  prioOneDefaultBackground: {
+    backgroundColor: theme.palette.secondary.main,
+  },
+  prioOneAccentBackground: {
+    borderLeftColor: theme.palette.secondary.light,
+  },
+}));
+
+type Props = { hubUrl: string | undefined };
+
+/**
+ * Adds a custom background to the hub, if one is defined. Currently only PrioOne has a custom background.
+ * Mind that this component determines itself whether and which background to render based on the hubUrl
+ * and the current pathname. This means that the background is not customizable by the user.
+ * Some backgrounds are limited to certain pages (e.g. auth pages or browse pages).
+ * Additionally, mobile versions are not supported yet and will result in no background being rendered.
+ * @param hubUrl The URL of the hub.
+ *
+ */
+export default function CustomBackground({ hubUrl }: Props) {
+  const mobileScreenSize = useMediaQuery((theme: Theme) => theme.breakpoints.down("md"));
+
+  // TODO: mobileScreenSize is not yet supported
+  if (!hubUrl || mobileScreenSize) {
```

Comment:
Is mobile screen size not yet supported because it is too complicated or because designs are missing?